### PR TITLE
Binary

### PR DESCRIPTION
**Pull request by dwightguth**
**please review**

### CODE REVIEW

**Comment by dwightguth** [comment]
Jenkins: test this please

**Comment by dwightguth** [comment]
Jenkins: test this please

**Comment by dwightguth** [comment]
Jenkins: test this please

**Comment by dwightguth** [comment]
@chathhorn please review

File: scripts/kcc [hunk 1]
```diff
@@ -480,9 +488,17 @@ sub getKRunCommand {
...
+               # wrap $thisObj with `unwrapObj`()
+               push(@objTexts, "$thisObj\x02\x00\x00\x00\x00\x00\x00\x00\x09\x00u\x00n\x00w\x00r\x00a\x00p\x00O\x00b\x00j\x00\x00\x00\x00\x01");
           }
-          my $objText = join('~>', @objTexts);
+          my $objText = join('', @objTexts);
+          my $one = chr((scalar @objTexts >> 24) & 0xff);
+          my $two = chr((scalar @objTexts >> 16) & 0xff);
+          my $three = chr((scalar @objTexts >> 8) & 0xff);
+          my $four = chr((scalar @objTexts >> 0) & 0xff);
+          $objText = "\x7fKAST\x04\x00\x01$objText\x03$one$two$three$four\x07";
```
Comment (chathhorn) [inline scripts/kcc]: If this ugliness isn't just a temporary hack, did you consider pack? E.g. for the length of the list:
pack("N", scalar @objTexts)

The rest of the PR looks good!

**Comment by chathhorn** [comment]
Merging this so things are fixed.

**Pull request merged**